core/storage/beam_job/gae_models.py

```diff
@@ -201,7 +201,11 @@ def get_new_id(cls) -> str:  # type: ignore[override]
 
     @staticmethod
     def get_deletion_policy() -> base_models.DELETION_POLICY:
-        """Model doesn't contain any data directly corresponding to a user."""
+        """Model doesn't contain any data directly corresponding to a user.
+
+        This model is marked as deleted 24 weeks after its creation.
```

Comment:
Perhaps this should just indicate the reference function and not the time (otherwise the two places might be difficult to keep in sync if we tweak the deletion numbers in cron_services).

Ditto for UserQueryModel and JobModel.

Reply:
Done.